docs/DesignDocs/RemoteAttestationCollaterals.md

```diff
@@ -0,0 +1,285 @@
+Remote Attestation Collaterals
+=====
+
+Adding attestation APIs to support collaterals for remote attestation.  Collaterals are information from a second source that the user can provide for attestation verification.
+
+Motivation
+----------
+
+Currently, the existing collaterals used for Intel SGX quote verification are not exposed to the user.  Adding these new APIs allows the user to specify these collaterals instead of using the default collaterals used by OE SDK.
+
+
+User Experience
+---------------
+
+There are 3 user scenarios.
+
+### 1. User does not specify the collaterals:
+
```

Comment:
Could you add some more description on each user scenario before getting to API?

Reply:
Done.